iphone/layout/layout/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,68 +1,117 @@
 {
   "images" : [
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "filename" : "DefaultIcon-Dark.png",
+      "idiom" : "universal",
+      "platform" : "ios",
+      "size" : "1024x1024"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "tinted"
+        }
+      ],
+      "filename" : "DefaultIcon-Tinted.png",
```

Comment:
Does this require additional files by default or is it fully backwards compatible? How does the generated Xcode project handle their absence?

Reply:
We should have no issue because in the _build.js the -dark and -tinted icons have `minXcodeVer: '16.0'` so older versions should skip it and if the icon is not given the os makes its own version.